pages/linux/btrfs-subvolume.md

```diff
@@ -0,0 +1,28 @@
+# btrfs subvolume
+
+> Manage btrfs subvolumes and snapshots.
+> More information: <https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-subvolume>.
+
+- Create a new empty subvolume:
+
+`sudo btrfs subvolume create {{path/to/new_subvolume}}`
+
+- List all subvolumes and snapshots in the specified filesystem:
+
+`sudo btrfs subvolume list {{path/to/btrfs_filesystem}}`
+
+- Delete a subvolume:
+
+`sudo btrfs subvolume delete {{path/to/subvolume}}`
+
+- Create a read-only snapshot of an existing subvolume:
+
+`sudo btrfs subvolume snapshot -r {{path/to/source_subvolume}} {{path/to/target}}`
```

Comment:
Not sure if this is correct...
```suggestion
`sudo btrfs subvolume snapshot --rootid {{path/to/source_subvolume}} {{path/to/target}}`
```

Reply:
It isn't, actually. That would be the long form of `-r`, yes - but the `-r` for `show` and not `snapshot`. For the snapshot subsubcommand, it's short for "read-only", rather than "rootid".

Reply:
I though maybe so.